UKHO.PeriodicOutputService/UKHO.BESS.ConfigurationService/Validation/ConfigValidator.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using FluentValidation;
+using FluentValidation.Results;
+using NCrontab;
+using UKHO.PeriodicOutputService.Common.Enums;
+using UKHO.PeriodicOutputService.Common.Models.Bess;
+
+namespace UKHO.BESS.ConfigurationService.Validation
+{
+    public interface IConfigValidator
+    {
+        ValidationResult Validate(BessConfig configurationSetting);
+    }
+
+    public class ConfigValidator : AbstractValidator<BessConfig>, IConfigValidator
+    {
+        public ConfigValidator()
+        {
+            RuleFor(config => config.Name)
+               .Must(name => !string.IsNullOrEmpty(name?.Trim()))
+               .WithMessage("Attribute is missing or value not provided")
```

Comment:
As this validation message text is same for different validation can we make constant string and use that across ?
```suggestion
               .WithMessage("Attribute is missing or value is not provided")
```

Reply:
Resolved. Thanks